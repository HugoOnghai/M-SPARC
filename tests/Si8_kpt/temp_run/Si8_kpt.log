
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat Si8_kpt.inpt
# nprocs: 24

# Test: Si8
LATVEC:
0.5 0.5 0.0
0.0 0.5 0.5
0.5 0.0 0.5
CELL: 7.297341981845170 7.297341981845170 7.297341981845170
MESH_SPACING: 0.20
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
EXCHANGE_CORRELATION: LDA_PW
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.775131
TOL_SCF: 1e-6
CALC_PRES: 1
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
MIXING_VARIABLE: potential
MIXING_PRECOND: none
<\INPT>

 Reading .ion file ...
 Number of atom types : 1
 Total number of atoms: 2
 Default atomic mass for Si is 28.085000
 pseudo_dir # 1 = Si.psp8

 COORD:
   0.072973419818452   0.087568103782142   0.080270761800297
   1.824335495461292   1.824335495461292   1.824335495461292

   0.030168927102176

 kpoint grid before symmetry:
   0.215255956306122   0.215255956306122   0.215255956306122
  -0.215255956306122   0.215255956306122   0.215255956306122
   0.215255956306122  -0.215255956306122   0.215255956306122
  -0.215255956306122  -0.215255956306122   0.215255956306122
   0.215255956306122   0.215255956306122  -0.215255956306122
  -0.215255956306122   0.215255956306122  -0.215255956306122
   0.215255956306122  -0.215255956306122  -0.215255956306122
  -0.215255956306122  -0.215255956306122  -0.215255956306122

 kpoint grid after symmetry:
   0.215255956306122   0.215255956306122  -0.215255956306122
  -0.215255956306122   0.215255956306122  -0.215255956306122
   0.215255956306122  -0.215255956306122  -0.215255956306122
  -0.215255956306122  -0.215255956306122  -0.215255956306122

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 10
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (10.236 sec)

 Estimated memory usage:
 Total:  244.19 MB
 orbitals            :  113.91 MB
 sparse matrices     :  120.31 MB
 global-size vectors :    4.98 MB
 mixing histories    :    4.98 MB

 Finding rb for Si ...
 rb = {3.851 3.851 3.851}, int_b = -4.000034115107398, err_rb = 3.412e-05
 rb = {4.054 4.054 4.054}, int_b = -4.000032573043000, err_rb = 3.257e-05
 rb = {4.257 4.257 4.257}, int_b = -4.000024897748159, err_rb = 2.490e-05
 rb = {4.459 4.459 4.459}, int_b = -4.000024897748159, err_rb = 2.490e-05
 rb = {4.662 4.662 4.662}, int_b = -4.000020192401200, err_rb = 2.019e-05
 rb = {4.865 4.865 4.865}, int_b = -4.000016780735490, err_rb = 1.678e-05
 rb = {5.068 5.068 5.068}, int_b = -4.000015517777813, err_rb = 1.552e-05
 rb = {5.270 5.270 5.270}, int_b = -4.000013684590516, err_rb = 1.368e-05
 rb = {5.473 5.473 5.473}, int_b = -4.000013677321020, err_rb = 1.368e-05
 rb = {5.676 5.676 5.676}, int_b = -4.000012040022384, err_rb = 1.204e-05
 rb = {5.878 5.878 5.878}, int_b = -4.000012132044415, err_rb = 1.213e-05
 rb = {6.081 6.081 6.081}, int_b = -4.000012194640414, err_rb = 1.219e-05
 rb = {6.284 6.284 6.284}, int_b = -4.000013488224029, err_rb = 1.349e-05
 rb = {6.487 6.487 6.487}, int_b = -4.000010800711335, err_rb = 1.080e-05
 rb = {6.689 6.689 6.689}, int_b = -4.000011252210280, err_rb = 1.125e-05
 rb = {6.892 6.892 6.892}, int_b = -4.000009812907582, err_rb = 9.813e-06
 rb = {7.095 7.095 7.095}, int_b = -4.000009763123072, err_rb = 9.763e-06
 rb = {7.297 7.297 7.297}, int_b = -4.000009924218288, err_rb = 9.924e-06
 rb = {7.500 7.500 7.500}, int_b = -4.000001082097663, err_rb = 1.082e-06
 rb = {7.703 7.703 7.703}, int_b = -3.999999936241774, err_rb = 6.376e-08
 rb = {7.905 7.905 7.905}, int_b = -4.000000015814123, err_rb = 1.581e-08
 rb = {8.108 8.108 8.108}, int_b = -3.999999994584763, err_rb = 5.415e-09
 rb = {8.108 8.108 8.108}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 7.999999999728

 Integration b_ref = 8.000000000020

 ****************************************
 *          Eself_ref = 57.199451       *
 ****************************************
 Done. (3.335245 s)
 Time for b calculation: 3.350 seconds.

 Starting calculating nonlocal projectors ...
 Done. (1.935156 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 9.174e-09 in 46 iterations.
 Poisson problem took 3.676532s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 5.327882
 This SCF iteration took 82.696 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.689884
 This SCF iteration took 41.421 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = 0.307092
 This SCF iteration took 41.555 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = 0.207938
 AAR converged to a relative residual of 8.49594e-09 in 37 iterations.
 Poisson problem took 3.385608s
 Escc = 0.16315425
 Etot = -7.90092749
 Eatom = -3.95046375
 Error in SCF iteration: 2.6728e-02 
 This SCF iteration took 45.097 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = 0.202424
 AAR converged to a relative residual of 2.75383e-09 in 37 iterations.
 Poisson problem took 2.948283s
 Escc = 0.06446099
 Etot = -7.92384534
 Eatom = -3.96192267
 Error in SCF iteration: 1.5866e-02 
 This SCF iteration took 44.542 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = 0.206045
 AAR converged to a relative residual of 7.927e-09 in 32 iterations.
 Poisson problem took 3.215209s
 Escc = 0.01359953
 Etot = -7.92593550
 Eatom = -3.96296775
 Error in SCF iteration: 5.0249e-03 
 This SCF iteration took 45.192 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = 0.207040
 AAR converged to a relative residual of 2.74181e-09 in 31 iterations.
 Poisson problem took 3.072653s
 Escc = -0.00133630
 Etot = -7.92615643
 Eatom = -3.96307821
 Error in SCF iteration: 9.3860e-04 
 This SCF iteration took 46.427 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = 0.206914
 AAR converged to a relative residual of 6.80982e-09 in 25 iterations.
 Poisson problem took 2.447056s
 Escc = -0.00023318
 Etot = -7.92617376
 Eatom = -3.96308688
 Error in SCF iteration: 2.3333e-04 
 This SCF iteration took 44.559 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = 0.206910
 AAR converged to a relative residual of 9.85186e-09 in 21 iterations.
 Poisson problem took 2.018637s
 Escc = -0.00008691
 Etot = -7.92617543
 Eatom = -3.96308772
 Error in SCF iteration: 1.0256e-04 
 This SCF iteration took 43.973 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = 0.206900
 AAR converged to a relative residual of 6.69289e-09 in 20 iterations.
 Poisson problem took 1.922086s
 Escc = 0.00005207
 Etot = -7.92617560
 Eatom = -3.96308780
 Error in SCF iteration: 2.0761e-05 
 This SCF iteration took 42.725 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = 0.206908
 AAR converged to a relative residual of 2.76422e-09 in 19 iterations.
 Poisson problem took 1.855948s
 Escc = -0.00004138
 Etot = -7.92617560
 Eatom = -3.96308780
 Error in SCF iteration: 1.1030e-05 
 This SCF iteration took 43.108 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = 0.206905
 AAR converged to a relative residual of 9.69871e-09 in 16 iterations.
 Poisson problem took 1.540223s
 Escc = -0.00000039
 Etot = -7.92617554
 Eatom = -3.96308777
 Error in SCF iteration: 3.3061e-06 
 This SCF iteration took 42.908 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = 0.206905
 AAR converged to a relative residual of 6.05037e-09 in 13 iterations.
 Poisson problem took 1.258472s
 Escc = -0.00000280
 Etot = -7.92617561
 Eatom = -3.96308781
 Error in SCF iteration: 1.0464e-06 
 This SCF iteration took 42.582 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = 0.206904
 AAR converged to a relative residual of 1.77717e-09 in 13 iterations.
 Poisson problem took 1.277478s
 Escc = 0.00000142
 Etot = -7.92617560
 Eatom = -3.96308780
 Error in SCF iteration: 4.1323e-07 
 This SCF iteration took 42.868 s.


 Finished SCF iteration in 11 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -7.926175597 Ha.       *
 *          Energy per atom = -3.963087799 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 3.960 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
  -0.011501322775373  -0.012143980464402  -0.012795415366023
   0.011501322775373   0.012143980464402   0.012795415366023

 Max magnitude of forces (Ha/Bohr):   0.021058949201994


 Time for calculating forces: 5.416575 s.

["Stress in GPa"


]   1.605214379520425   5.068605961916993   5.158359400278376
   5.068605961916993   1.615987809770216   5.246830761563538
   5.158359400278376   5.246830761563538   1.627564292209750


 Time for calculating stress: 16.949585 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  0.072973 	  0.087568 	  0.080271 
  1.824335 	  1.824335 	  1.824335 

 Run-time of the program: 693.902916 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.707106781186547
                        L1: 7.297341981845170
                        L2: 7.297341981845170
                        L3: 7.297341981845170
                        Nx: 36
                        Ny: 36
                        Nz: 36
                         N: 46656
                      ecut: -1
              mesh_spacing: 0.200000000000000
                   kptgrid: [4x3 double]
                  kptshift: [1x3 double]
                      nkpt: [2 2 2]
                     tnkpt: 4
                      wkpt: [4x1 double]
                        BC: 2
                       BCx: 0
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 10
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157751310000000e+02
                       bet: 9.999996636976061e+02
                       npl: 35
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 0
                 Nelectron: 8
                     n_typ: 1
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 4.108888888888888e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 1
            PrintForceFlag: 1
          PrintAtomPosFlag: 0
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'Si8_kpt'
            inputfile_path: '.'
                        XC: 'LDA_PW'
                       Atm: [1x1 struct]
                     n_atm: 2
                     Atoms: [2x3 double]
           mvAtmConstraint: [2x3 double]
                    IsFrac: 1
                    IsSpin: 0
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [2x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.202703943940144
                        dy: 0.202703943940144
                        dz: 0.202703943940144
                        dV: 0.005889407407407
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [46656x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [468x1 logical]
                 isOutr_11: [468x1 logical]
                      I_11: [468x1 double]
                     II_11: [468x1 double]
                      V_11: [468x1 double]
                 isOutl_22: [468x1 logical]
                 isOutr_22: [468x1 logical]
                      I_22: [468x1 double]
                     II_22: [468x1 double]
                      V_22: [468x1 double]
                 isOutl_33: [468x1 logical]
                 isOutr_33: [468x1 logical]
                      I_33: [468x1 double]
                     II_33: [468x1 double]
                      V_33: [468x1 double]
                  isOutl_1: [432x1 logical]
                  isOutr_1: [432x1 logical]
                       I_1: [432x1 double]
                      II_1: [432x1 double]
                       V_1: [432x1 double]
                  isOutl_2: [432x1 logical]
                  isOutr_2: [432x1 logical]
                       I_2: [432x1 double]
                      II_2: [432x1 double]
                       V_2: [432x1 double]
                  isOutl_3: [432x1 logical]
                  isOutr_3: [432x1 logical]
                       I_3: [432x1 double]
                      II_3: [432x1 double]
                       V_3: [432x1 double]
                 G_JOutl_1: [559872x1 logical]
                 G_JOutr_1: [559872x1 logical]
                     G_J_1: [559872x1 double]
                     G_I_1: [559872x1 double]
                     G_V_1: [559872x1 double]
                 G_JOutl_2: [559872x1 logical]
                 G_JOutr_2: [559872x1 logical]
                     G_J_2: [559872x1 double]
                     G_I_2: [559872x1 double]
                     G_V_2: [559872x1 double]
                 G_JOutl_3: [559872x1 logical]
                 G_JOutr_3: [559872x1 logical]
                     G_J_3: [559872x1 double]
                     G_I_3: [559872x1 double]
                     G_V_3: [559872x1 double]
                   Lap_std: [46656x46656 double]
                    grad_1: [46656x46656 double]
                    grad_2: [46656x46656 double]
                    grad_3: [46656x46656 double]
                LapPreconL: [46656x46656 double]
                LapPreconU: [46656x46656 double]
              memory_usage: 256048136
               staticfname: 'Si8_kpt.static'
                 suffixNum: 0
                  outfname: 'Si8_kpt.out'
                  parallel: 0
                         b: [46656x1 double]
                     b_ref: [46656x1 double]
                    rho_at: [46656x1 double]
                     Eself: 25.548993941219511
                 Eself_ref: 57.199450578353364
                       V_c: [46656x1 double]
                 PosCharge: 7.999999999727667
                 NegCharge: -7.999999999727667
                    E_corr: -4.258301078152726e-05
                       rho: [46656x1 double]
                      Atom: [1x2 struct]
                       phi: [46656x1 double]
                       Vxc: [46656x1 double]
                 dvxcdgrho: [46656x1 double]
                      Veff: [46656x1 double]
                         X: [46656x7 double]
                         F: [46656x7 double]
          mixing_hist_fkm1: [46656x1 double]
          mixing_hist_xkm1: [46656x1 double]
                       psi: [46656x10x4 double]
    upper_bound_guess_vecs: [46656x4 double]
                    EigVal: [10x4 double]
                  lambda_f: 0.206904361130471
                       occ: [10x4 double]
                    Etotal: -7.926175597335892
                     Eband: 0.181887895966598
                       Exc: -2.389098461957905
                    Exc_dc: -3.116415128295567
                  Eelec_dc: 16.713654947590914
                      Eent: -6.247259646137534e-12
                      Escc: 1.417005471770667e-06
                   S_Debug: [1x1 struct]
                     force: [2x3 double]
                 abs_force: [2x1 double]
                    Stress: [3x3 double]
                  Pressure: -1.616255493833464
             stress_dimred: [3x3 double]

