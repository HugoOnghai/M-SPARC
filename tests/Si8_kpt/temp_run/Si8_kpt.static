Atomic forces (Ha/Bohr):
     -0.0115013228      -0.0121439805      -0.0127954154
      0.0115013228       0.0121439805       0.0127954154
Stress (GPa):
  1.6052143795E+00   5.0686059619E+00   5.1583594003E+00 
  5.0686059619E+00   1.6159878098E+00   5.2468307616E+00 
  5.1583594003E+00   5.2468307616E+00   1.6275642922E+00 
