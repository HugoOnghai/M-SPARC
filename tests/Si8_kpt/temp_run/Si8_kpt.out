***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:09:21                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.297342 7.297342 7.297342 
LATVEC:
0.500000000000000 0.500000000000000 0.000000000000000 
0.000000000000000 0.500000000000000 0.500000000000000 
0.500000000000000 0.000000000000000 0.500000000000000 
FD_GRID: 36 36 36
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
KPOINT_SHIFT: 5.000000e-01 5.000000e-01 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 35
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si8_kpt.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.202704 (Bohr)
Number of symmetry adapted k-points:  4
Output printed to                  :  Si8_kpt.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.11 8.11 8.11
Number of atoms of type 1          :  2
Estimated total memory usage       :  244.19 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9504637472E+00        2.673E-02        210.766
2            -3.9619226725E+00        1.587E-02        44.540
3            -3.9629677506E+00        5.025E-03        45.189
4            -3.9630782135E+00        9.386E-04        46.424
5            -3.9630868784E+00        2.333E-04        44.556
6            -3.9630877160E+00        1.026E-04        43.971
7            -3.9630877982E+00        2.076E-05        42.722
8            -3.9630877987E+00        1.103E-05        43.105
9            -3.9630877713E+00        3.306E-06        42.907
10           -3.9630878070E+00        1.046E-06        42.580
11           -3.9630877987E+00        4.132E-07        42.865
Total number of SCF: 11    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9630877987E+00 (Ha/atom)
Total free energy                  : -7.9261755973E+00 (Ha)
Band structure energy              :  1.8188789597E-01 (Ha)
Exchange correlation energy        : -2.3890984620E+00 (Ha)
Self and correction energy         : -2.5549036524E+01 (Ha)
Entropy*kb*T                       : -6.2472596461E-12 (Ha)
Fermi level                        :  2.0690436113E-01 (Ha)
Average force                      :  2.1058949202E-02 (Ha/Bohr)
Maximum force                      :  2.1058949202E-02 (Ha/Bohr)
Time for force calculation         :  5.417 (sec)
Pressure                           : -1.6162554938E+00 (GPa)
Max stress                         :  5.2468307616E+00 (GPa)
Time for stress calculation        :  16.952 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  693.903 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
